app/services/cert/rhsm_client.rb

```diff
@@ -10,7 +10,9 @@ def initialize(cert)
     end
 
     def uuid
-      drop_cn_prefix_from_subject(@cert.subject.to_s)
+      # rubocop:disable Performance/Detect
```

Comment:
Could this be written in a way to avoid disabling the cop?

Also, if I call .uuid multiple times this could be quite slow. Maybe you can use some memoization to put the value into an instance variable